src/lib/viewers/text/TextBaseViewer.js

```diff
@@ -1,9 +1,12 @@
+import React from 'react';
 import BaseViewer from '../BaseViewer';
 import Controls from '../../Controls';
+import ControlsRoot from '../controls';
+import TextControls from './TextControls';
 import ZoomControls from '../../ZoomControls';
-import { checkPermission } from '../../file';
 import { CLASS_IS_PRINTABLE, CLASS_IS_SELECTABLE, PERMISSION_DOWNLOAD } from '../../constants';
 import { ICON_FULLSCREEN_IN, ICON_FULLSCREEN_OUT } from '../../icons/icons';
+import { checkPermission } from '../../file';
```

Comment:
Intentional?

Reply:
My editor auto-sorted it this way. I assume capital letters come first.